abel/direct.py

```diff
@@ -240,9 +223,9 @@ def _pyabel_direct_integral(f, r, correction, int_func=simpson_rule_wrong):
 
 
 # append the same docstring to all functions
-iabel_direct.__doc__ += _direct_doctsting
-fabel_direct.__doc__ += _direct_doctsting
-_abel_transform_wrapper.__doc__ += _direct_doctsting
+#iabel_direct.__doc__ += _direct_doctsting
+#fabel_direct.__doc__ += _direct_doctsting
+direct_transform.__doc__ += _direct_doctsting
```

Comment:
Is it now just one function? So should the docstring just live at the top of the function as normal?